app/build.gradle

```diff
@@ -12,8 +12,8 @@ android {
         resValue "string", "app_name", "NewPipe"
         minSdkVersion 19
         targetSdkVersion 28
-        versionCode 870
-        versionName "0.18.7"
+        versionCode 900
```

Comment:
Why increase by 30? It was a coincidence that e.g. 0.18.7 was 870 and it should just be increased by 10 like always imho. However now it's already too late.

Reply:
Why though? It's nice to be consistent. In fact, I would prefer 1910 for 0.19.1 and eventually, 10000 for 1.0.0. It's just symbolic.

Reply:
Yeah. I agree. 
I think we can do this when going 1.0.0 which should be soon after #2907 is merged

Reply:
Copy-paste of my argument on IRC:

>With his 10000 example it's still not possible to give a "correct" version code for e.g. 1.0.10. I don't think we should have ridiculously long version codes, though that doesn't really matter since we probably won't hit the 32-bit limit that soon. Also, version codes are purely for Android to check whether an APK is newer than the currently installed one. Version codes aren't for users to see, that's what version names are for. Google says: " Typically, you would release the first version of your app with versionCode set to 1, then monotonically increase the value with each release, regardless of whether the release constitutes a major or minor release. This means that the versionCode value does not necessarily have a strong resemblance to the app release version that is visible to the user (see versionName, below). Apps and publishing services should not display this version value to users." Iirc TheAssassin once argued that we shouldn't just increase by 1 every time, but by 10, which he had a reason for, so we switched to that. Imho it just doesn't make sense to somehow tie it to the version names.

Reply:
It's much easier and foreseeable IMO to just increase by the ever same amount. The interval of 10 isn't even really needed any more, but we should keep it. As @wb9688 showed, any "binding" to a version code doesn't work, there's always cases in which it breaks. A version code is 3 integers, not 1.